### DISCUSSION TITLE
FormScope还没做完

### DISCUSSION THREAD

**Discussion by danatechgithub4** [junerver/ComposeHooks#7] [Q&A] [answered]
Email和Regex的Validator未实现，自定义的Validator如何加入呢？

**Comment by junerver** [comment]
> Email和Regex的Validator未实现，自定义的Validator如何加入呢？

确实，因为这个库是为我们产品提供支持的，这个feat的优先级比较低，合并的比较草率，后续有时间我会继续完善它，如果你有好的构思欢迎pr

**Comment by junerver** [comment, marked as answer]
> > Email和Regex的Validator未实现，自定义的Validator如何加入呢？
> 
> 确实，因为这个库是为我们产品提供支持的，这个feat的优先级比较低，合并的比较草率，后续有时间我会继续完善它，如果你有好的构思欢迎pr

1.0.14 快照版本，完成了基本的一些校验器，增加了自定义校验器，欢迎体验
```grovvy
maven { url 'https://s01.oss.sonatype.org/content/repositories/snapshots' }

implementation("xyz.junerver.compose:hooks:1.0.14-SNAPSHOT")
```
